torch_geometric/profile/utils.py

```diff
@@ -153,7 +153,7 @@ def get_gpu_memory_from_ipex(
     max_allocated = byte_to_megabyte(max_allocated, digits)
     max_reserved = byte_to_megabyte(max_reserved, digits)
     max_active = byte_to_megabyte(max_active, digits)
-    ipex.xpu.reset_peak_memory_stats()
+    ipex.xpu.reset_peak_memory_stats(device)
```

Comment:
It looks like we don't test this much even before this PR, but it could be nice to add a quick test case maybe 😄

Reply:
I think `reset_peak_memory_stats()` is well-tested in IPEX. If we would like to test `get_gpu_memory_from_ipex` per device, it would require CI environment with at least two XPU devices - I'm not sure it's doable.